packages/nextjs/hooks/scaffold-stark/useSwitchNetwork.ts

```diff
@@ -1,15 +1,27 @@
+
+const getChainId = (network: string): string => {
+  switch (network) {
+    case "mainnet":
+      return "SN_MAIN";
+    case "devnet":
+      return "SN_GOERLI";
+    default:
+      return `SN_${network.toUpperCase()}`;
+  }
+};
+
 export const useSwitchNetwork = () => {
   return {
     switchNetwork: async (network: string) => {
       // @ts-ignore
```

Comment:
Please help to refactor to `@ts-expect-error` so that we can enforce why we need to skip the intepreter there, thank you!